cookbooks/aws-parallelcluster-install/recipes/users.rb

```diff
@@ -0,0 +1,22 @@
+# Setup cluster admin group
```

Comment:
Missed copyright, license, recipe name etc. i'd suggest you to do this patch in 2 commits, the first one just to rename cluster_admin_user to users and the second one to add your changes

Reply:
gotcha, will fix